### ISSUE TITLE
20.10: Encryption prompt freezes at boot

### ISSUE THREAD

**Issue by leviport** [pop-os/plymouth-theme#18]
When rebooting with external display connected, the encryption prompt usually seems frozen. Sometimes I see just the gray background with nothing on it, and sometimes I see the dialog, but the cursor does not blink and character placeholder symbols do not appear when I type. 

Typing the password and hitting enter still unlocks the drive and takes me to GDM. Hitting Escape once shows me the non-graphical decryption prompt, and hitting Escape once more re-displays a normal and working encryption prompt.

I believe this freeze is what prevents the prompt from appearing on the external display, whether the lid is closed or not.

I occasionally see the prompt display correctly on both displays, but I've only seen it twice out of ~30 reboots. 

The problem does not seem to happen if I fully shut the machine down and power it back on with the power button.

I believe I've seen this happen a couple times on my darp5 with proprietary firmware, but it seems more rare.

```
system76@pop-os:~$ apt policy plymouth-theme-pop-basic plymouth
plymouth-theme-pop-basic:
  Installed: 2.0.1~1602194015~20.10~000f1ca
  Candidate: 2.0.1~1602194015~20.10~000f1ca
  Version table:
 *** 2.0.1~1602194015~20.10~000f1ca 1001
       1001 http://ppa.launchpad.net/system76/pop/ubuntu groovy/main amd64 Packages
       1001 http://ppa.launchpad.net/system76/pop/ubuntu groovy/main i386 Packages
        100 /var/lib/dpkg/status
plymouth:
  Installed: 0.9.5-0ubuntu2.1
  Candidate: 0.9.5-0ubuntu2.1
  Version table:
 *** 0.9.5-0ubuntu2.1 500
        500 http://us.archive.ubuntu.com/ubuntu groovy-updates/main amd64 Packages
        100 /var/lib/dpkg/status
     0.9.5-0ubuntu2 500
        500 http://us.archive.ubuntu.com/ubuntu groovy/main amd64 Packages
```

**Comment by leviport** [comment]
Editing the topic, as this seems to be happening on more than just the pang10

**Comment by jacobgkau** [comment]
Forgot to hit send on this comment...

When this happens on oryp7:
- Pressing `Esc` once shows the text prompt.
- Pressing `Esc` a second time shows the themed prompt.
- While typing the password, I was sent back to the text prompt, which then displayed the message `error: unexpectedly disconnected from boot status daemon`.

![IMG_20210223_093316__01](https://user-images.githubusercontent.com/7199422/108875446-6f80a200-75ba-11eb-9b8d-f413472f3d83.jpg)

**Comment by leviport** [comment]
I believe this was fixed with the Plymouth rewrite in #28
